src/tokens.ts

```diff
@@ -93,6 +93,7 @@ export const TOKEN_SYMBOLS_MAP = {
     symbol: "ETH",
     decimals: 18,
     addresses: {
+      [CHAIN_IDs.ALEPH_ZERO]: "0xD648529D4803d3467bA8850577BEd4e4b8Ae583C",
```

Comment:
This address seems to be USDT: https://evm-explorer.alephzero.org/address/0xD648529D4803d3467bA8850577BEd4e4b8Ae583C

Reply:
Done - this was a copy paste issue during a pair programming session - the corrected version was on local 😓. Thanks for the callout!